src/python/gudhi/datasets/generators/_points.cc

```diff
@@ -122,4 +153,34 @@ PYBIND11_MODULE(_points, m)
 
           If sample is 'grid': (⌊n_samples**(1./dim)⌋**dim, 2*dim), where shape[0] is rounded down to the closest perfect 'dim'th power.
           )pbdoc");
+
+  m.def("c3dtorus",
+        &generate_points_on_3d_torus,
+        py::arg("n_samples"),
+        py::arg("R"),
+        py::arg("r"),
+        R"pbdoc(
+          Generate random i.i.d. points on a 2-torus in R^3
+
+          :param n_samples: The number of points to be generated.
+          :type n_samples: integer
+          :param R: The big radius.
+          :type R: float
+          :param r: The small radius.
+          :type r: float
+          :returns: the generated points on a torus.
+
+          The typical donut shape is obtained with an aspect ratio (R/r) of the torus equal to 3/2.
+          )pbdoc");
+
+  m.def("seed",
```

Comment:
It looks inconvenient to hide it in a submodule. Why not at the top level, like `gudhi.set_random_seed`, or maybe `gudhi.random.seed`? Otherwise it looks like we may have to set one seed for the generators, one seed for the internal use of subsampling, one seed for the internal use of mapper, etc, and then we might as well pass a generator argument.